core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\HttpCache;
+
+use FOS\HttpCache\SymfonyCache\CacheEvent;
+use FOS\HttpCache\SymfonyCache\Events;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class StripCookiesSubscriber implements EventSubscriberInterface
+{
+    private const BLACKLIST = [
+        // Modals are always for JS only
+        '^(.*)?modal(.*)?$',
+
+        // Google Analytics (https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage)
+        '^_ga$',
+        '^_gid$',
+        '^_gat$',
+        '^AMP_TOKEN$',
+        '^_gac_.+$',
+
+        // Matomo (https://matomo.org/faq/general/faq_146/)
+        '^_pk_id$',
+        '^_pk_ref$',
+        '^_pk_ses$',
+        '^_pk_cvar$',
+        '^_pk_hsr$',
+    ];
```

Comment:
Here are some more cookies that are shipped with Contao:

 * `TS_TABLE_` (used in the `tablesort.js` file)
 * `BE_PAGE_OFFSET`
 * `TL_VIEW`

I have already seen your ticket https://github.com/contao/contao/issues/459 hereto, however, we cannot be sure that none of the extensions use those cookies, therefore we should blacklist them.

Also, as briefly discussed on Slack, we should blacklist the `__cfduid` cookie used by Cloudflare.

Reply:
* `TS_TABLE_` - we cannot ignore that cookie because it would break the functionality. The cookie would never make it to the application if we ignore it. That's another candidate for localstorage then.
* `BE_PAGE_OFFSET` - same here, we cannot ignore as it would break back end scroll to functionality. As mentioned in https://github.com/contao/contao/issues/459 it needs to be reworked to use localstorage or at least restricted to `/contao` so it's not in the FE. But we cannot ignore it as it would not get to the application then.
* `TL_VIEW` - same here again, if we would ignore it, the mobile layout functionality would be broken. You don't have any cache anyway if you use the `mobile-layout-bundle` so there's no point in ignoring it.

I'll add `__cfduid` though.